### PR TITLE
Make header title link inline

### LINKED ISSUE #2665: Make vis titles "inline" instead of block
Sometimes users want to select a visualization and they open it instead because the whole red area (where the title is placed) is clickable:

screen shot 2015-03-09 at 14 31 40

PS: we'll probably need to use 'inline-flex' to avoid breaking the .PermissionIndicator that appears with shared maps.

### PR DESCRIPTION
**Pull request by javierarce**
Fixes #2665 

Sometimes users want to select a visualization and they open it instead because the whole red area (where the title is placed) is clickable:

screen shot 2015-03-09 at 14 31 40

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2148/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2190/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2214/))

**Comment by javierarce** [comment]
@CartoDB/frontend: can you review this fix, please?

**Comment by javierarce** [comment]
retest this, please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2215/))

**Comment by xavijam** [comment]
Have you checked "affected-visualizations" items as well?

File: app/assets/stylesheets/new_variables/mixins.css.scss [hunk 1]
```diff
@@ -34,6 +34,11 @@
   display: -webkit-flex;     /* NEW - Chrome */
   display: flex;             /* NEW, Spec - Opera 12.1, Firefox 20+ */
 }
+@mixin display-inline-flex() {
+  display: inline-block;
+  display: -webkit-inline-flex;
+  display: inline-flex;
```
Comment (xavijam) [inline app/assets/stylesheets/new_variables/mixins.css.scss]: I feel this is not compatible with rest of browsers, unfortunately you should add the css3 prefixes.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2244/))

**Comment by javierarce** [comment]
@xavijam: you mean the links in the dialog that appears when you delete a dataset. Yes, this works there too.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2247/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2289/))

**Comment by alonsogarciapablo** [comment]
I just came across this same issue:

captura de pantalla 2015-04-01 a las 15 47 22

@javierarce I think it's not fixed yet.

**Comment by alonsogarciapablo** [comment]
Oh, I just realized you reverted the fix...

**Comment by javierarce** [comment]
You're totally right. It was reverted because I merged when they were making the onpremise build and then I forgot about it. I'll deploy it next monday.

**Pull request merged**